src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Comment = (
+  {
+    commentId,
+    commentBody,
+    removeComment,
+    commentErrorId,
+  },
+) => (
```

Comment:
не переноси скобки объекта

```suggestion
export const Comment = (
    commentId,
    commentBody,
    removeComment,
    commentErrorId,
  }) => (
```